### DISCUSSION TITLE
How to calculate back yhat<i> using the components breakdown

### DISCUSSION THREAD

**Discussion by deepanshu650** [ourownstory/neural_prophet#1223] [Q&A - forecasting best practices] [answered]
Hi all,

I want to know how to recalculate lets say yhat1, using the different components breakdown we get after prediction.
I am using AR net, lagged and future regressor in multiplicative setting, and yearly, monthly, weekly seasonality and some external events all in multiplicative setting.

Also i do see columns - event_multiplicative, event_name_of_first_event, event_name_of_second_event and so on...
what is the relation between these columns, as in how are the individual event columns combined to get final event column.
same thing if i pass multiple future_regressors

**Comment by LeonieFreisinger** [comment, marked as answer]
@deepanshu650 
Feel free to have a look at the NP paper on page 4
https://arxiv.org/pdf/2111.15397.pdf

**Reply by deepanshu650** [reply]
@LeonieFreisinger thanks for your reply

I was going through the dataset in the this example https://neuralprophet.com/tutorials/tutorial03.html.

If I use seasonality in additive mode, then i am able to get back yhat1 using 
yhat1 = trend + season_yearly + season_weekly

In case seasonality is in multiplicative mode, then I was using
yhat1 = trend*(1 + season_yearly + season_weekly)

but in this formulation I was unable to match the yhat1, given by `model.predict()`  dataframe

Can you help me what is going wrong in this formulation.
